.github/workflows/tests.yml

```diff
@@ -42,7 +42,8 @@ jobs:
         with:
           go-version: 1.17
       - run: |
```

Comment:
Add a name like "Run all tests" would be good.

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsname